api/renter_test.go

```diff
@@ -1525,7 +1541,7 @@ func TestRenterPricesHandlerPricey(t *testing.T) {
 // from low quality hosts when there are higher quality hosts available.
 func TestContractorHostRemoval(t *testing.T) {
 	// Create a renter and 2 hosts. Connect to the hosts and start uploading.
-	if testing.Short() {
+	if testing.Short() || !build.VLONG {
```

Comment:
Moved to vlong mainly because it was flaking on appveyor for reasons I really didn't understand. Without access to the logs that the renter creates, there was little chance of debugging it. I'll be run continuously on our other server though, so we can use that to make sure this test is working as desired, or to fix any NDFs that the server picks up.